docs/cody/core-concepts/token-limits.mdx

```diff
@@ -10,41 +10,46 @@ All other models are currently capped at **7,000 tokens** of shared context betw
 
 Here's a detailed breakdown of the token limits by model:
 
-
-
 <Tabs>
   <Tab title="Free">
```

Comment:
@abeatrix token limits are also updated for the Free tier. Can you please take one final pass on the numbers? 🙏